src/Aspire.ProjectTemplates/templates/aspire-starter/.template.config/template.json

```diff
@@ -84,12 +84,26 @@
       "displayName": "_Use Redis for caching (requires a supported container runtime)",
       "description": "Configures whether to setup the application to use Redis for caching. Requires a supported continer runtime to run locally, see https://aka.ms/dotnet/aspire/containers for more details."
     },
-    "CreateTestsProject": {
+    "TestProjectFrameworkKind": {
```

Comment:
Any suggestion for a better name? Maybe `SelectedTestFramework`?

Reply:
It doesn't really matter, this isn't visible to users. Just make sure it makes sense for us the authors, e.g. `TestFramework`.